addons/create-cluster-admin/create-cluster-admin.addon

```diff
@@ -0,0 +1,5 @@
+# Name: create-cluster-admin
+# Description: Create admin user and add cluster-admin role to it.
+
+oc create user admin --full-name=admin
```

Comment:
@LalatenduMohanty No we agreed about only the addon name not about `user` and why you think user should be `cluster-admin` ?